csclient/csclient_test.go

```diff
@@ -1429,3 +1440,15 @@ func (s *suite) TestWhoAmI(c *gc.C) {
 	c.Assert(err, gc.IsNil)
 	c.Assert(response.User, gc.Equals, "bob")
 }
+
+func (s *suite) setPublic(c *gc.C, id *charm.URL) {
```

Comment:
Define it on a common embedded test struct?
Or just as a function receiving a client?